model/src/main/java/org/projectnessie/model/BaseMergeTransplant.java

```diff
@@ -44,11 +44,24 @@ public interface BaseMergeTransplant {
   MergeBehavior getDefaultKeyMergeMode();
 
   @Nullable
+  @JsonInclude(Include.NON_NULL)
   Boolean isDryRun();
 
   @Nullable
+  @JsonInclude(Include.NON_NULL)
   Boolean isFetchAdditionalInfo();
 
+  /**
+   * When set to {@code true}, the {@link org.projectnessie.api.TreeApi#mergeRefIntoBranch(String,
+   * String, Merge)} and {@link org.projectnessie.api.TreeApi#transplantCommitsIntoBranch(String,
+   * String, String, Transplant)} operations will return {@link MergeResponse} object when a content
+   * based conflict cannot be resolved, instead of throwing a {@link
+   * org.projectnessie.error.NessieReferenceConflictException}.
```

Comment:
tangential: it feels that we may want to have a dedicated exception for content conflicts in merges rather than `NessieReferenceConflictException`... but it's beyond the scope of this PR for sure.